### ISSUE TITLE
Access token popup reappears every time I double-click in a hovercard?

### ISSUE THREAD

**Issue by aminomancer** [Justineo/github-hovercard#164]
Hi there, love this extension but I'm wondering if this is intentional or if there's something wrong with my storage. This is on firefox 86 on windows 10 fwiw. Extension version 1.9.3. So I already input a token and saved it, but any time I double-click inside a hovercard, the popup comes back. I had assumed it would only pop up on double-clicking the first time or if the token expires, but maybe that's just my bias. But if so it's pretty irritating since I will frequently double or triple click on text to select it so I can copy it. Seems like it would be better to just add a section to the options document, or only attach the click listener if there's no valid token. So I'm not sure if this is a bug report or feature request but thanks anyway

Oh and it's definitely on triple click, not double click. Kinda hard to tell with my trackpad

**Comment by Justineo** [comment]
It should only pop up on triple clicks and it’s intentional. If it popped out on double clicks then it’s a bug.

**Comment by aminomancer** [comment]
Yeah I'm pretty sure it was just on triple clicks and I was clicking too fast. I still think itd make more sense if it stopped asking for an access token after you input one though, but I'll try to make a PR to that end
